### PR TITLE
Admin config menu updates

### LINKED ISSUE #2795: As a webmaster, I can easily navigate the admin page to successfully configure my site.
# Story

As a webmaster, I can easily navigate the admin page to successfully configure my site.

# Motivation
Our custom modules that have configuration pages are named differently in the admin section. Some are prefixed with SiteNow, UIowa, University of Iowa, etc. They are also all located in the System menu section which seems wrong for things like SiteNow Articles which should probably be under Content Authoring.

Screen Shot 2021-01-06 at 8 52 06 AM

IMO, the average person doesn't need to see a prefix at all and Articles Configuration would be better than SiteNow/UI/Uiowa Articles.

# Proposed solution

Decide on a naming convention and change existing admin menu links for custom modules. Also place custom module menu links in the appropriate menu parent.

This issue does **not** include changing module machine names.

### PR DESCRIPTION
**Pull request by e-marie-w**
Fixes #2795 

# How to test
Pull branch, sync a site, get site running
Go to Configuration menu (site/admin/config)
Admire how all our custom modules are in their own category at the top left of the page. No more scrolling or ctrl-f (or at least less). No more wondering why some are SiteNow and some UIowa, the prefixes have been removed completely. 
Click into some modules and note the url path is correctly /admin/config/sitenow/ for them. 
Ignore the fact the rest of the path and the page name itself still contains the prefixes as this issue does not address those.
Feel free to suggest better descriptions, still not 100% on what some of the modules do.

### CODE REVIEW

File: docroot/modules/custom/sitenow_dcv/sitenow_dcv.links.menu.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 sitenow_dcv.settings:
-  title: 'SiteNow DCV'
+  title: 'DCV'
```
Comment (richardbporter) [inline docroot/modules/custom/sitenow_dcv/sitenow_dcv.links.menu.yml]: I think this should be spelled out as Domain Control Validation.

Comment (e-marie-w) [reply docroot/modules/custom/sitenow_dcv/sitenow_dcv.links.menu.yml]: Should the description stay spelled out too, or do we want to switch to abbreviating that?

**Comment by joewhitsitt** [comment]
Alerts test needs to be updated with new path

```
@api
  Scenario: Create a custom alert                                                           # features/Alerts.feature:9
    Given I am logged in as a user with role webmaster                                      # Drupal\FeatureContext::iAmLoggedInAsUserWithRole()
    And I am on "/admin/config/system/uiowa-alerts"
```

https://github.com/uiowa/uiowa/blob/main/tests/behat/features/Alerts.feature#L11

File: docroot/modules/custom/sitenow_dcv/sitenow_dcv.links.menu.yml (hunk 1 above)
Comment (richardbporter) [reply docroot/modules/custom/sitenow_dcv/sitenow_dcv.links.menu.yml]: I think its fine how it is now.

**Comment by joewhitsitt** [comment]
Haven't tested yet, but with these changes we will need to update our docs. https://sitenow.uiowa.edu/search?terms=admin/config/system/

same with our tours...
https://github.com/uiowa/uiowa/search?p=1&q=%22admin%2Fconfig%2Fsystem%2F%22

what about site specific things like transportation calculator?

**Comment by richardbporter** [comment]
Yea sorry I overlooked the path changes. :(

**Comment by e-marie-w** [comment]
Ack definitely read this Friday and then forgot about it.

> Haven't tested yet, but with these changes we will need to update our docs. https://sitenow.uiowa.edu/search?terms=admin/config/system/

Do I update the docs now, or wait until the PR is actually ready/released?

> same with our tours...https://github.com/uiowa/uiowa/search?p=1&q=%22admin%2Fconfig%2Fsystem%2F%22

Ohh so this is fixing other places where admin/config/system exists and needs to be updated. Was confused by the tours thing, now I see it's on the later pages of that list.

> what about site specific things like transportation calculator?

This I don't really have a clue about - I can definitely update specific things, I'm just not sure how many we have.

**Comment by joewhitsitt** [comment]
>Do I update the docs now, or wait until the PR is actually ready/released?

I think once it is merged, we should make an effort.

Yeah, https://github.com/uiowa/uiowa/search?p=1&q=%22admin%2Fconfig%2Fsystem%2F%22 will highlight the other bits of code/config we should consider. Some site splits (e.g. transportation) have their own settings pages. I think moving these to SiteNow as well would be appropriate.

**Comment by e-marie-w** [comment]
Okay, successfully updated `admin/config` paths from `/system` to `/sitenow` for the transportation calculator, the tours files, and a few other files that referenced it. I left alone any `/system` referencing basic site settings since we didn't move that at this point. 

I also updated the path for articles and events in sitenow_v2 but now I'm wondering if that was necessary or not. 

The search from earlier only found the transportation calculator, could sites being using a different category than `/system` for site-specific modules or are we good with just that search?

**Comment by joewhitsitt** [comment]
Thanks for the additions. I will pull and test

**Comment by e-marie-w** [comment]
Forgot to ask in last comment where to find the tours thing is for testing purposes. Good news is I found it on my own, bad news is I don't think they're working correctly yet.

Edit: Nevermind they do work, just needed to run cim instead of cr since they're config files.

**Comment by joewhitsitt** [comment]
If you want to switch it from draft to review and request approval, I will approve.

Tours, transportation, etc. all seem to work for me. I might be a proponent of a "Site Custom" category in the future but that is for another day.

**Review by joewhitsitt — APPROVED** [review]

**Pull request merged**